en/SQLDatabase.md

```diff
@@ -7,3 +7,42 @@ title: Shared SQL Database
 JabRef is able to support collaborative work using a shared SQL database.
 This feature is available since JabRef 3.6.
 If you used to work with an older version of JabRef, please refer to [SQLDatabaseMigration](SQLDatabaseMigration) for information how to update your data.
+
+## Usage
+
+To use this feature you have to connect to a remote database. 
+To do so you have to open **File** in the menu bar and then click the **Open remote database** item.
+The **Open remote database** dialog will open and you will have to fill in the remote's database connection settings.
+Under the field **Database type** you can choose between PostgreSQL, MySQL and Oracle depending on your remote database.
+Then, you have to fill out the rest of the fields with the according information.
+
+![Screenshot of Open shared database dialog](./images/open-shared-database-dialog.png)
```

Comment:
I did it and tried it out locally. The word terminated just stands in the end, is this intended?